src/V3Split.cpp

```diff
@@ -248,12 +252,18 @@ class SplitReorderBaseVisitor VL_NOT_FINAL : public VNVisitor {
     const VNUser4InUse m_inuser4;
 
 protected:
+    using LhsRevisions = std::unordered_map<const AstVarScope*, int>;
```

Comment:
```suggestion
    using LhsRevisions = std::map<const AstVarScope*, int>;
```

Clearing a unordered_map takes on order of milliseconds, so isn't something you want to do each block.

For that matter can't we use a userp?  Much faster.

Or I wonder if this can be tracked using only the graph.